files/project_team/IFCTunnelDep2.1.ifc

```diff
@@ -0,0 +1,604 @@
+#210= IFCPRODUCTDEFINITIONSHAPE($,$,(#211));
+#211= IFCSHAPEREPRESENTATION(#41,'Axis','Segment',(#212));
+#212= IFCCURVESEGMENT(.CONTSAMEGRADIENTSAMECURVATURE.,#213,IFCNONNEGATIVELENGTHMEASURE(0.0),IFCPARAMETERVALUE(91.146858467386),#216);
+#213= IFCAXIS2PLACEMENT2D(#214,#215);
+#214= IFCCARTESIANPOINT((27738.979156,91.708871));
+#215= IFCDIRECTION((0.999992,0.00399997));
+#216= IFCLINE(#217,#218);
+#217= IFCCARTESIANPOINT((0.0,0.0));
+#218= IFCVECTOR(#219,1.0);
+#219= IFCDIRECTION((0.999992,0.00399997));
+#220= IFCALIGNMENTVERTICALSEGMENT($,$,27738.979156,91.146129,91.708871,0.00400000265238435,0.00400000265238435,$,.CONSTANTGRADIENT.);
+#221= IFCALIGNMENTSEGMENT('3nsQwxjrH74x_TSdV6PmdG',$,'E',$,$,#222,#225,#235);
+#222= IFCLOCALPLACEMENT($,#223);
+#223= IFCAXIS2PLACEMENT3D(#224,$,$);
+#224= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#225= IFCPRODUCTDEFINITIONSHAPE($,$,(#226));
+#226= IFCSHAPEREPRESENTATION(#41,'Axis','Segment',(#227));
+#227= IFCCURVESEGMENT(.CONTSAMEGRADIENTSAMECURVATURE.,#228,IFCPARAMETERVALUE(1.57479627989573),IFCPARAMETERVALUE(-0.0104998358636363),#231);
+#228= IFCAXIS2PLACEMENT2D(#229,#230);
+#229= IFCCARTESIANPOINT((27830.125285,92.073456));
+#230= IFCDIRECTION((0.999992,0.00399994));
+#231= IFCCIRCLE(#232,11000.0);
+#232= IFCAXIS2PLACEMENT2D(#233,#234);
+#233= IFCCARTESIANPOINT((43.999367,-10999.912002));
+#234= IFCDIRECTION((1.0,0.0));
+#235= IFCALIGNMENTVERTICALSEGMENT($,$,27830.125285,115.497574,92.073456,0.003999974433555,-0.00649997430106127,-11000.0,.CIRCULARARC.);
+#236= IFCALIGNMENTSEGMENT('1xqIhY0Vb4Ou$rlTMboPRl',$,'E',$,$,#237,#240,#250);
+#237= IFCLOCALPLACEMENT($,#238);
+#238= IFCAXIS2PLACEMENT3D(#239,$,$);
+#239= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#240= IFCPRODUCTDEFINITIONSHAPE($,$,(#241));
+#241= IFCSHAPEREPRESENTATION(#41,'Axis','Segment',(#242));
+#242= IFCCURVESEGMENT(.CONTSAMEGRADIENTSAMECURVATURE.,#243,IFCNONNEGATIVELENGTHMEASURE(0.0),IFCPARAMETERVALUE(68.7470059419781),#246);
+#243= IFCAXIS2PLACEMENT2D(#244,#245);
+#244= IFCCARTESIANPOINT((27945.622859,91.929088));
+#245= IFCDIRECTION((0.99997888,-0.00649987));
+#246= IFCLINE(#247,#248);
+#247= IFCCARTESIANPOINT((0.0,0.0));
+#248= IFCVECTOR(#249,1.0);
+#249= IFCDIRECTION((0.99997888,-0.00649987));
+#250= IFCALIGNMENTVERTICALSEGMENT($,$,27945.622859,68.745554,91.929088,-0.00650000252063129,-0.00650000252063129,$,.CONSTANTGRADIENT.);
+#251= IFCALIGNMENTSEGMENT('23E5X3Xab2vBbrFcdBux6z',$,'E',$,$,#252,#255,#265);
+#252= IFCLOCALPLACEMENT($,#253);
+#253= IFCAXIS2PLACEMENT3D(#254,$,$);
+#254= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#255= IFCPRODUCTDEFINITIONSHAPE($,$,(#256));
+#256= IFCSHAPEREPRESENTATION(#41,'Axis','Segment',(#257));
+#257= IFCCURVESEGMENT(.CONTSAMEGRADIENTSAMECURVATURE.,#258,IFCPARAMETERVALUE(1.5642963819142),IFCPARAMETERVALUE(-0.002299793167857),#261);
+#258= IFCAXIS2PLACEMENT2D(#259,#260);
+#259= IFCCARTESIANPOINT((28014.368413,91.482242));
+#260= IFCDIRECTION((0.99997888,-0.0064999));
+#261= IFCCIRCLE(#262,28000.0);
```

Comment:
28km for a radius is quite big. Please double-check.

Reply:
They must use big radius:es in the vertical alignment for rail. From the LandXML:
         <CircCurve radius="-10000.0000000" length="185.8518247">
              27050.062737 84.264410
          </CircCurve>
          <CircCurve radius="17000.0000000" length="127.4875501">
              27675.235592 91.453897
          </CircCurve>
          <CircCurve radius="11000.0000000" length="115.4981945">
              27887.874451 92.304453
          </CircCurve>
          <CircCurve radius="28000.0000000" length="64.3942087">
              28046.564851 91.272965
          </CircCurve>
          <CircCurve radius="-25000.0000000" length="25.0052981">
              28535.289674 86.972187
          </CircCurve>
          <CircCurve radius="20000.0000000" length="22.0040455">
              29007.234710 83.291165
          </CircCurve>

Reply:
I see. Thank you for the snippets!